leetcode/01-two-sum.java

```diff
@@ -0,0 +1,17 @@
+package leetcode;
+import java.util.*;
+class Solution {
+    public int[] twoSum(int[] nums, int target) {
+        int[] res = new int[2];
+        HashMap<Integer,Integer> map = new HashMap<>();
+        for(int i=0; i<nums.length; i++){
+            if(map.containsKey(target - nums[i])){
+                res[0] = map.get(target - nums[i]);
+                res[1] = i;
+                break;
+            }
+            map.put(nums[i], i);
+        }
+        return res;
+    }
+}
```

Comment:
Please add an empty line at EOF to satisfy github

Reply:
resolved